### PR TITLE
feat: add kiva classic style to causes settings components

### PR DESCRIPTION
**Pull request by eddieferrer**
GD-192
Screen Shot 2021-12-16 at 1 52 20 PM
Screen Shot 2021-12-16 at 1 52 11 PM

### CODE REVIEW

File: src/components/AutoDeposit/AutoDepositDropInPaymentWrapper.vue [hunk 1]
```diff
@@ -10,16 +10,14 @@
...
 				<kv-button
 					class="tw-mb-2"
-					value="submit"
-					id="dropin-submit"
-					:state="!enableConfirmButton || submitting ? 'disabled' : ''"
+					:state="buttonState"
 					@click="submitDropInAutoDeposit"
 				>
 					<kv-icon name="lock" />
-					Confirm <kv-loading-spinner v-if="submitting" />
```
Comment (eddieferrer) [inline src/components/AutoDeposit/AutoDepositDropInPaymentWrapper.vue:22]: I think we can remove these in place spinner in favor of the button states

File: src/components/Forms/CausesUpdateForm.vue [hunk 2]
```diff
@@ -95,7 +93,7 @@
 					<strong>Your contribution will:</strong>
 				</div>
 				<div class="column">
-					<kv-select v-model="form.categoryId" class="group-dropdown">
+					<kv-select v-model="form.categoryId" class="group-dropdown" id="causes-categories-select">
```
Comment (eddieferrer) [inline src/components/Forms/CausesUpdateForm.vue:96]: KvSelect requires id prop

Comment (ryan-ludwig) [reply src/components/Forms/CausesUpdateForm.vue:96]: The goal with the required id is to help remind that all inputs need `<label>` elements. I think you could use the "Your contribution will", or make a new `<label class="tw-sr-only" for="causes-category-select">Some descriptive label of the select</label>` nearby

**Review by ryan-ludwig — APPROVED** [review]
Small thing with the label, otherwise looks great! Thanks for doing this

**Pull request merged**